sources/packages/backend/apps/queue-consumers/src/processors/assessment/_tests_/cancel-application-assessment.processor.e2e-spec.ts

```diff
@@ -489,6 +493,98 @@ describe(
       );
     });
 
+    it(
+      "Should not assess for impacted application to create reassessment when an application assessment with " +
+        "a declined COE is being cancelled.",
+      async () => {
+        // Arrange
+        // Create the student to be shared across all these applications.
+        const student = await saveFakeStudent(db.dataSource);
+
+        // Current application to be cancelled with a declined COE.
+        const currentApplicationToCancel =
+          await saveFakeApplicationDisbursements(
+            db.dataSource,
+            { student },
+            {
+              offeringIntensity: OfferingIntensity.partTime,
+              applicationStatus: ApplicationStatus.Cancelled,
+              currentAssessmentInitialValues: {
+                assessmentWorkflowId: "some fake id",
+                assessmentDate: new Date(),
+                studentAssessmentStatus:
+                  StudentAssessmentStatus.CancellationQueued,
+              },
+              firstDisbursementInitialValues: { coeStatus: COEStatus.declined },
+            },
+          );
+        const futureImpactedApplication =
+          await saveFakeApplicationDisbursements(
+            db.dataSource,
+            { student },
+            {
+              offeringIntensity: OfferingIntensity.partTime,
+              applicationStatus: ApplicationStatus.Completed,
+              currentAssessmentInitialValues: {
+                assessmentWorkflowId: "some fake id",
+                assessmentDate: addDays(1, new Date()),
```

Comment:
Minor, not a blocker, one more if you would like to change to `addDays(1, new Date())`.

Reply:
Modified here and other places of the file.